html/css/panel.css

```diff
@@ -483,4 +483,20 @@ input.search:focus  {
 
 .download-text p {
 
+}
+
+
+#reset-list-button-div,
```

Comment:
It would be better to use a class for styling in this instance; both elements would have this class and their respective ids